RELEASE-NOTES.md

```diff
@@ -1,5 +1,32 @@
 # Prefect Release Notes
 
+## Release 2.7.10
+
+### Enhancements
+- Add [`is_schedule_active` option](https://docs.prefect.io/api-ref/prefect/deployments/#prefect.deployments.Deployment) to `Deployment` class to allow control of automatic scheduling — https://github.com/PrefectHQ/prefect/pull/7430
+- Add documentation links to blocks in UI — https://github.com/PrefectHQ/prefect/pull/8210
+- Add Kubernetes kube-system permissions to Prefect agent template for retrieving UUID from kube-system namespace — https://github.com/PrefectHQ/prefect/pull/8205
+- Enhance security of secrets to blocks in UI by incorporating secret fields in blocks' schema — https://github.com/PrefectHQ/prefect/pull/8246
+- Add consolidation and manual dispatch of multi-arch Docker build and publish logic — https://github.com/PrefectHQ/prefect/pull/7902
+
+### Experimental
+- Add functionality to specify a work pool when starting an agent — https://github.com/PrefectHQ/prefect/pull/8222
+- Disable `Work Queues` tab view when work pools are enabled — https://github.com/PrefectHQ/prefect/pull/8257
+- Fix property for `WorkersTable` in UI — https://github.com/PrefectHQ/prefect/pull/8232
+
+### Documentation
+- [Add Prefect Cloud Quickstart tutorial](https://docs.prefect.io/ui/cloud-getting-started/) — https://github.com/PrefectHQ/prefect/pull/8227
+- Add [prefect-openai](https://prefecthq.github.io/prefect-openai/) to [Collections catalog](https://docs.prefect.io/collections/catalog/) — https://github.com/PrefectHQ/prefect/pull/8236
```

Comment:
Should be in a `### Collections` section — see other releases for typical format of this message.